src/App.css

```diff
@@ -3,3 +3,305 @@
 *::after {
   box-sizing: border-box;
 }
+
+.Container {
+  background-color: rgb(249, 249, 249);
+}
+
+.NavBar {
+  background-color: rgb(255, 255, 255);
+}
+
+.ListGroup {
+  margin: 10px 0 10px 0;
+  border: solid !important;
+  border-width: 1px !important;
+  border-color: rgb(220, 220, 220) !important;
+  border-radius: 4px;
+  height: 170px;
+  display: flex;
```

Comment:
- [ ] Nice one here. I notice the `display: flex` was repeated about 7x in your CSS. This negates the DRY principle for CSS best practices. I will suggest you create a separate class for it or create a variable and reuse the variable instead of reusing the property. Example

You can have this `root`at the top of your CSS 
```
:root {
  --flex: flex;
}
```
Then you can reuse the variable inside the CSS class or element or ID like so
```
.ListGroup {
  other properties
  display: var(--flex)
}
```
In this case, we are only declaring the CSS property once. And then reusing the variable throughout our CSS.
There are other better methods. I will encourage you to use the one that works best for you.

Reply:
Thank you, really appreciated that comment Sir.